README.md

```diff
@@ -45,11 +45,9 @@ file_path_or_url = 'path/to/file.mp4'
 # process of fixing this though.
 player = OMXPlayer(file_path_or_url)
 
-# The player will initially be paused
-
-player.play()
-sleep(5)
 player.pause()
```

Comment:
@longagofaraway ah, thanks for the commit reference, I was mis-remembering the default.

Going to review and probably merge tonight :)